test/lib/runners/baseRunner.js

```diff
@@ -68,10 +54,40 @@ module.exports = class BaseRunner {
             return;
           }
 
-          if (output.includes(snippet.expected)) {
-            resolve();
-            return;
+          const
+            expected = Array.isArray(snippet.expected) ? snippet.expected : [snippet.expected];
```

Comment:
Why don't you use this npm package instead ? https://www.npmjs.com/package/is_array

Reply:
https://www.youtube.com/watch?v=oavMtUWDBTM